doc/move_group_python_interface/move_group_python_interface_tutorial.rst

```diff
@@ -22,7 +22,7 @@ Open two shells. Start RViz and wait for everything to finish loading in the fir
   roslaunch panda_moveit_config demo.launch
 
 Now run the Python code directly in the other shell using ``rosrun``.
-Note in some instances you may need to make the python script executable: ::
+Note that you may need to make the Python script executable (using  ``chmod +x /path_to_file/file_name.py``): ::
```

Comment:
The repository already tracks execution permissions and I can't find a python file without execution permissions in the repository. So I think this is the wrong place for this remark altogether.

But if we keep it nonetheless, I'm happy to add the additional explanation.

I'll deliberately not vote here and leave the decision to the next reviewer.

Reply:
I think that's a fair argument too, and I'm good with either of those outcomes. 

@Praveendwivedi did you have trouble executing the file or did the comment send you on this Google search for no reason?

Reply:
Thanks @v4hn & @felixvd  for the reviews. I didn't try the tutorial myself, I was just going through the tutorial and came across that comment. Since previously I've also faced such issues, I thought it would be better to include the command too.

 As @v4hn clearly pointed out since the execution permissions are already tracked and no such issue is found, the whole comment seems to be redundant to me. What do you think @felixvd ? 

I'm not sure if such issue may arise in the future though